Makefile

```diff
@@ -339,7 +339,7 @@ kind-image-build: kind image-build
 
 .PHONY: test-integration
 test-integration: manifests fmt vet envtest ginkgo ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" \
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" \
```

Comment:
I was getting the following error in my env. 

unable to make sure store is initialized: unable to make sure base binaries dir exists: mkdir /.local: permission denied